Documentation/guides/building-apps/build-properties.md

```diff
@@ -633,6 +639,11 @@ APK root directory. The format of the path is `lib\ARCH\wrap.sh` where
 + `x86_64`
 + `x86`
 
+## AndroidInstallModules
```

Comment:
This should go into `build-items.md`, not `build-properties.md`.  It's an item group, not a property.